packages/orchestrator-ui-components/src/rtk/endpoints/streamMessages.ts

```diff
@@ -66,17 +66,18 @@ const streamMessagesApi = orchestratorApi.injectEndpoints({
                     updateCachedData(() => false);
                 };
 
-                const invalidateTag = (tag: string) => {
-                    const tagToInvalidate = tag as CacheTags;
-                    if (validCacheTags.includes(tagToInvalidate)) {
+                const invalidateTag = (
+                    tag: CacheTags | { type: CacheTags; id: string },
```

Comment:
`{ type: CacheTags; id: string }` would be a nice one to make a type alias of

Reply:
Done